scripts/sdk/swagger.json

```diff
@@ -581,7 +581,8 @@
               "minimum": 1,
               "multipleOf": 1
             },
-            "in": "query"
+            "in": "query",
+            "name": "TODO"
```

Comment:
There is no query parameter in project so you can completely remove it.

Reply:
Thanks. I removed it.

Reply:
Entire `"paremeters"` property you can remove, I mean the following

```
,
        "parameters": [
          {
            "schema": {
              "type": "number",
              "minimum": 1,
              "multipleOf": 1
            }
          }
        ]
```

Reply:
Thanks again. I removed entire parameters property.

There's another missing name field in `public-data-create` api.
Should I set `xc-password` here?

As I checked, `public-data-create` will call dataInsert and `xc-password` header is checked inside it.
https://github.com/nocodb/nocodb/blob/0e5a4d64c7be957f68256bf250d838ead34dff87/packages/nocodb/src/lib/meta/api/publicApis/publicDataApis.ts#L449-L455
https://github.com/nocodb/nocodb/blob/0e5a4d64c7be957f68256bf250d838ead34dff87/packages/nocodb/src/lib/meta/api/publicApis/publicDataApis.ts#L185

Reply:
Yes, you can add that as a header and which is optional.

Reply:
Fixed.